api-report/container-runtime.api.md

```diff
@@ -718,16 +719,18 @@ export type SummarizerStopReason =
  | "disposed";
 
 // @public (undocumented)
-export class SummarizingWarning extends LoggingError implements ISummarizingWarning {
-    constructor(errorMessage: string, logged?: boolean);
+export class SummarizingWarning extends LoggingError implements ISummarizingWarning, IFluidErrorBase {
+    constructor(errorMessage: string, fluidErrorCode: string, logged?: boolean);
```

Comment:
This is a breaking change, no?

Reply:
So this summarizingWarning is not crossing the boundaries, it is in runtime only. So I don't think it is a breaking change.